From 297774fa4d01156c0327d6e6380a7ecae30bf875 Mon Sep 17 00:00:00 2001
From: Patrick Ohly <patrick.ohly@intel.com>
Date: Mon, 23 Mar 2015 15:01:39 -0700
Subject: [PATCH 1/2] cynara-db-migration: abort on errors

"set -e" enables error checking for all commands invoked by the script.
Previously, errors were silently ignored.
---
 migration/cynara-db-migration | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/migration/cynara-db-migration b/migration/cynara-db-migration
index ff9bd61..f6e7f94 100644
--- a/migration/cynara-db-migration
+++ b/migration/cynara-db-migration
@@ -19,6 +19,8 @@
 # @brief       Migration tool for Cynara's database
 #
 
+set -e
+
 ##### Constants (these must not be modified by shell)
 
 STATE_PATH='/var/cynara'
-- 
1.8.4.5

